cog.c

```diff
@@ -277,6 +281,10 @@ on_handle_local_options (GApplication *application,
         }
     }
 
+    if (s_options.web_extensions_dir != NULL) {
+        cog_shell_set_web_extensions_dir (shell, s_options.web_extensions_dir);
```

Comment:
I think this should use the existing `cog_shell_get_web_context()` accessor instead:

```c
webkit_web_context_set_web_extensions_directory (cog_shell_get_web_context (shell),
                                                 s_options.web_extensions_dir);
```

This way we avoid adding unneeded public API to the `CogShell` class.